lib/util/http/types.ts

```diff
@@ -34,3 +34,16 @@ export interface RequestStats {
 }
 
 export type OutgoingHttpHeaders = Record<string, string | string[] | undefined>;
+
+export interface GraphqlVariables {
+  [k: string]: unknown;
+}
```

Comment:
```suggestion
export type GraphqlVariables = Record<string, unknown>;
```
I think we should use record instead

Reply:
Won't work well with our graphql call

Reply:
My first attempt was to make it like Record, but this caused further complicated type errors

Reply:
Strange, shouldn't those types equivalent. 🤔

Reply:
It had something to do with (indexing) type of the record keys

Reply:
So, I've decided it's not the battle I would like to pick

Reply:
Ok, still strange but ok